app/views/conversations/show.html.erb

```diff
@@ -1,7 +1,14 @@
 <%= turbo_stream_from @conversation %>
-<div class="flex h-screen" data-controller="click-toggle" data-click-toggle-flippable-class="md:flex">
+<div class="flex h-screen"
+     data-controller="click-toggle"
+     data-click-toggle-flippable-class="md:flex"
+     data-action="keydown.meta+shift+s@document->click-toggle#handleEvent:prevent"
+     >
```

Comment:
I'll start trying to adopt this line-breaking since these HTML tags are all getting so long, thanks to the combo of stimulus and tailwind

Reply:
This line ... is like a line of Perl code. It's so obtuse to declare it all this way, but I like that I don't have to write import a bunch of node packages to make it work.